packages/graphback-core/src/scalars/index.ts

```diff
@@ -27,12 +28,18 @@ const { parseLiteral, parseValue, ...objectIDConfig } = extractConfig(ObjectIDRe
 export const GraphbackObjectID = new GraphQLScalarType({
   ...objectIDConfig,
   parseLiteral: (ast: ValueNode, variables: { [key: string]: any}) => {
-    const { ObjectID } = require('mongodb');
+    let ObjectIDFinal = ObjectID;
+    try {
+      ObjectIDFinal = require('bson-ext').ObjectID;
+    } catch {}
 
-    return new ObjectID(parseLiteral(ast, variables));
+    return new ObjectIDFinal(parseLiteral(ast, variables));
   },
   parseValue: (value: any) => {
-    const { ObjectID } = require('mongodb');
+    let ObjectIDFinal = ObjectID;
+    try {
+      ObjectIDFinal = require('bson-ext').ObjectID;
+    } catch {}
 
     return new ObjectID(parseValue(value));
```

Comment:
Aah this is my mistake, it should be `new ObjectIDFinal(...)`. 

The idea is to always prefer [bson-ext](https://github.com/mongodb-js/bson-ext) a C++ native implementation which is faster than [js-bson](https://github.com/mongodb/js-bson)

I'll create a followup PR for this.